examples/simple.js

```diff
@@ -11,12 +10,6 @@ var map = new ol.Map({
       source: new ol.source.OSM()
     })
   ],
-  controls: ol.control.defaults({
-    attributionOptions: /** @type {olx.control.AttributionOptions} */ ({
-      collapsible: false
-    })
-  }),
```

Comment:
I'm taking the very widespread use of collapsed attribution as a sign that it is acceptable.  If others have concern about this, I'm inclined to switch the base layer for this example to keep it simpler.